src/app.js

```diff
@@ -65,6 +72,9 @@ export default class App extends React.Component {
             <Route exact path="/" render={() => <p>Home</p>} />
           </div>
         </Router>
+        <Dialogs active={this.state.activeDialog}
```

Comment:
As mentioned above, better keeping App for higher level configs than specific components